### PR TITLE
Update install.md

### LINKED ISSUE #17: Minimum required size for USB key for installation ISO
Trying to follow the installation guide to create an installation media using a USB key, but don't know what size I need.
It would be nice if the install docs mentioned the minimum required size.
The ISO itself is well under 1GB, but I suspect the installer will be much larger.

### PR DESCRIPTION
**Pull request by JamuelStarkey**
Fixes #17

### CODE REVIEW

File: docs/install.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ You can download the 8.2 ISO here: <https://mirrors.xcp-ng.org/isos/8.2/xcp-ng-8
 SHA256 checksums, GPG signatures and net-install ISO are available [here](https://xcp-ng.org/#easy-to-install).
 
 
-Then, create the install media (e.g. USB key):
+Then, create the install media (e.g. a USB key 1GB or larger should work):
```
Comment (olivierlambert) [inline docs/install.md:16]: Would it be "an USB key"?

**Review by olivierlambert — CHANGES_REQUESTED** [review]

File: docs/install.md (hunk 1 above)
Comment (JamuelStarkey) [reply docs/install.md:16]: If you are referring to "a" vs "an" -- in proper English it would be "a".

Comment (olivierlambert) [reply docs/install.md:16]: That's why I used "would" because I wasn't sure. If you say it's "a", then it is :+1:

**Comment by olivierlambert** [comment]
Doc rebuilt and available :+1: Thanks for your contribution @JamuelStarkey

**Pull request merged**